cpp/src/arrow/ipc/read_write_test.cc

```diff
@@ -2565,11 +2565,14 @@ void GetReadRecordBatchReadRanges(
   const int32_t magic_size = static_cast<int>(strlen(ipc::internal::kArrowMagicBytes));
   // read magic and footer length IO
   auto file_end_size = magic_size + sizeof(int32_t);
+  auto footer_length_offset = buffer->size() - file_end_size;
+  auto footer_length = BitUtil::FromLittleEndian(
+      *reinterpret_cast<const int32_t*>(buffer->data() + footer_length_offset));
```

Comment:
use `util::SafeLoadAs`?

Reply:
Thanks.  I was not aware of that utility.  I switched to this.